app/controllers/application_controller.rb

```diff
@@ -1,10 +1,143 @@
 require './config/environment'
+require "./app/models/user"
 
 class ApplicationController < Sinatra::Base
 
+
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
   end
 
-end
\ No newline at end of file
+  configure do
+    enable :sessions
+    set :session_secret, "secret"
+  end
+
+  get '/' do
+    erb :index
+  end
+
+  get '/tweets' do
+    if !logged_in?
+      redirect to '/login'
+    end
+    @user = self.current_user
+    #binding.pry
+    erb :'/tweets/index'
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/new'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    #binding.pry
+    if !params[:content].empty?
+      @tweet = Tweet.create(content: params[:content], user_id: current_user.id)
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.